README.md

```diff
@@ -1,21 +1,97 @@
-# Fury EKS Installer
+<h1>
+    <img src="./docs/assets/fury_installer.png?raw=true" align="left" width="105" style="margin-right: 15px"/>
+    Fury EKS Installer
+</h1>
 
-**Installer** - *Managed Cluster Installer* - EKS
+![Release](https://img.shields.io/github/v/release/sighupio/fury-eks-installer?label=Latest%20Release)
+![License](https://img.shields.io/github/license/sighupio/fury-eks-installer?label=License)
+[![Slack](https://img.shields.io/badge/slack-@kubernetes/fury-yellow.svg?logo=slack&label=Slack)](https://kubernetes.slack.com/archives/C0154HYTAQH)
 
-![EKS](https://miro.medium.com/max/2978/1*5YAnP-tNi1Tkjzk1urIJYw.png)
+<!-- <KFD-DOCS> -->
 
-## What is this?
+**Fury EKS Installer** deploys a production-grade Kubernetes Fury cluster on Amazon Elastic Kubernetes Services (EKS).
 
-This repository contains the versioned installer/[terraform module](modules/eks) to seamless install an EKS cluster
-in AWS.
+If you are new to Kubernetes Fury please refer to the [official documentation][kfd-docs] on how to get started.
 
-## Documentation
+## Modules
 
-The updated installer documentation is available in the [Kubernetes Fury Distribution documentation][eks installer docs]
+The installer is composed of two different terraform modules:
 
-## License
+|            Module             |                  Description                   |
+| ----------------------------- | ---------------------------------------------- |
+| [VPC and VPN][vpc-vpn-module] | Deploy the necessary networking infrastructure |
+| [EKS][eks-module]             | Deploy the EKS cluster                         |
+
+Click on each module to see its full documentation.
+
+## Architecture
+
+The EKS installers deploys and configures a production-ready EKS cluster without having to learn all internals of the service.
+
+<img src="./docs/assets/fury_installer_architecture.png?raw=true" width="600" style="margin: 0 auto"/>
+
+The [EKS module][eks-module] deploys a **private control plane** cluster, where the control plane endpoint is not publicly accessible.
+
+The [VPC and VPN module][vpc-vpn-module] setups all the necessary networking infrastructure and a bastion host.
+
+The bastion host includes a OpenVPN instance easily manageable by using [furyagent][furyagent] to provide access to the cluster.
+
+> 🕵🏻‍♂️ [Furyagent][furyagent] is a tool developed by SIGHUP to manage OpenVPN and SSH user access to the bastion host.
+
+## Usage
```

Comment:
maybe we can point to the Quick Start guide here?

Reply:
Yes, this could be an idea. 
Ideally we should refactor the `furyagent` repository to explain the usage, and link all others installers to it.

I started doing some work but it's still WIP:
https://github.com/sighupio/furyagent/tree/feat/improve-docs

Not sure how to proceed tbh